vaadin-grid-pro-flow/src/main/java/com/vaadin/flow/component/gridpro/EditColumnConfigurator.java

```diff
@@ -47,9 +51,10 @@
      * @param column
      *            the column to edit, not <code>null</code>
      */
-    public EditColumnConfigurator(EditColumn<T> column) {
+    public EditColumnConfigurator(EditColumn<T> column, ValueProvider<T, ?> valueProvider) {
```

Comment:
Another breaking change. It looks like this constructor should never have been public, since users are not supposed to create this object on their own. I guess we can make it package-protected now as we are breaking stuff anyway. Please try to be careful in the future with the access modifiers, and don't expose any public API which is not supposed to be used by the user.

Reply:
Converted to be package scoped.